main/src/cgeo/geocaching/connector/gc/GCConstants.java

```diff
@@ -172,8 +172,8 @@
     final static Pattern PATTERN_USERTOKEN = Pattern.compile("userToken\\s*=\\s*'([^']+)'");
     final static Pattern PATTERN_LIST_PQ = Pattern.compile(Pattern.quote("(") + "(\\d+)" + Pattern.quote(")") + ".+?guid=(.+?)\".*?title=\"(.+?)\"");
 
-    // downloadable PQs,     <a href="/pocket/downloadpq.ashx?g=b1390cb5-88f3-4fcc-b5e9-f3786c30f773&src=web">
-    final static Pattern PATTERN_LIST_PQ_DL = Pattern.compile("downloadpq.ashx\\?g=([a-zA-Z0-9-]+).*?>\\s*(\\d+)\\s*</td>");
+    // downloadable PQs,
+    final static Pattern PATTERN_LIST_PQ_DL = Pattern.compile("downloadpq\\.ashx\\?g=([a-fA-F0-9-]+)[^>]*>\\s*([^<]*)<\\/a>\\s*<\\/td>\\s*<td[^>]*>[^<]*<\\/td>\\s*<td[^>]*>\\s*(\\d+)\\s*<\\/td>[^<]*<td>\\s*([^<(]*)(\\(([^\\s]+)[^<]*)?<");
```

Comment:
Parsing HTML with Regex isn't a good thing at all. I would prefer a DOM parser, but this would be a bigger refactoring.
I'll try to split it into smaller Patterns. Or should I make a proposal with a DOM parser?